contracts/axone-law-stone/src/contract.rs

```diff
@@ -1040,28 +1039,33 @@ mod tests {
     }
 
     #[test]
-    fn break_stone_admin() {
+    fn break_stone_creator() {
         let cases = vec![
-            ("not-admin", true, false, Some(ContractError::Unauthorized)),
-            ("not-admin", true, true, Some(ContractError::Unauthorized)),
-            ("admin", true, false, None),
-            ("anyone", false, false, None),
+            // creator, sender, broken, Error
+            (
+                "creator",
+                "sender",
+                false,
+                Some(ContractError::Unauthorized),
+            ),
+            ("creator", "sender", true, Some(ContractError::Unauthorized)),
+            ("creator", "creator", false, None),
+            ("creator", "sender", true, Some(ContractError::Unauthorized)),
```

Comment:
```suggestion
```

I think this one is written twice ? (L1051 : https://github.com/axone-protocol/contracts/pull/578/files#diff-54443171b7d0ca23f237f2fa2d378860cb912596d713dd09b70b8fd2988c92afR1051)

Reply:
Hey! Good catch. 😌  I've fixed it.